app/components/Feedback/feedback.css

```diff
@@ -70,3 +69,18 @@
   transform: translate(-9vw, var(--spacing-56));
   margin: var(--spacing-24);
 }
+
+.escape-feedback {
+  position: absolute;
+  top: 15px;
+  right: 15px;
+  line-height: 0;
+  border: none;
+  background: #fff;
+}
+
+@media (max-width: 780px) {
+  .feedback-form {
+    left: -20px;
```

Comment:
this looks suspicious - @melissasamworth ?

Reply:
haha yes, I didn't try very hard to properly align the feedback form on mobile, because I wanted to wait until the column sizing is fixed on mobile. I could have a crack at fixing the mobile column margin but I feel like @melissasamworth had something specific in mind?